cockroachcloud/export-logs.md

```diff
@@ -8,10 +8,6 @@ cloud: true
 
 {{ site.data.products.dedicated }} users can use the [Cloud API](cloud-api.html) to configure log export to [AWS CloudWatch](https://aws.amazon.com/cloudwatch/) or [GCP Cloud Logging](https://cloud.google.com/logging). Once the export is configured, logs will flow from all nodes in all regions of your {{ site.data.products.dedicated }} cluster to your chosen cloud log sink. You can configure log export to redact sensitive log entries, limit log output by severity, send log entries to specific log group targets by log channel, among others.
 
-{{site.data.alerts.callout_danger}}
-The {{ site.data.products.dedicated }} log export feature is only available on clusters created after August 11, 2022 (AWS) or September 9, 2022 (GCP).
```

Comment:
Note: the limitation was removed for **GCP only**. AWS still has this limitation. We're working on fixing this right now, but until it's merged, deployed, and run, the AWS limitation still exists.

Can we rework this to only remove the GCP bit, leaving the AWS piece?